// language-code/dist/index.interface.js
export var LanguageCode;
(function (LanguageCode) {
    /** Language code for Abkhazian */
    LanguageCode["ab"] = "ab";
    /** Language code for Afar */
    LanguageCode["aa"] = "aa";
    /** Language code for Afrikaans */
    LanguageCode["af"] = "af";
    /** Language code for Akan */
    LanguageCode["ak"] = "ak";
    /** Language code for Albanian */
    LanguageCode["sq"] = "sq";
    /** Language code for Amharic */
    LanguageCode["am"] = "am";
    /** Language code for Arabic */
    LanguageCode["ar"] = "ar";
    /** Language code for Aragonese */
    LanguageCode["an"] = "an";
    /** Language code for Armenian */
    LanguageCode["hy"] = "hy";
    /** Language code for Assamese */
    LanguageCode["as"] = "as";
    /** Language code for Avaric */
    LanguageCode["av"] = "av";
    /** Language code for Avestan */
    LanguageCode["ae"] = "ae";
    /** Language code for Aymara */
    LanguageCode["ay"] = "ay";
    /** Language code for Azerbaijani */
    LanguageCode["az"] = "az";
    /** Language code for Bambara */
    LanguageCode["bm"] = "bm";
    /** Language code for Bashkir */
    LanguageCode["ba"] = "ba";
    /** Language code for Basque */
    LanguageCode["eu"] = "eu";
    /** Language code for Belarusian */
    LanguageCode["be"] = "be";
    /** Language code for Bengali */
    LanguageCode["bn"] = "bn";
    /** Language code for Bihari languages */
    LanguageCode["bh"] = "bh";
    /** Language code for Bislama */
    LanguageCode["bi"] = "bi";
    /** Language code for Bosnian */
    LanguageCode["bs"] = "bs";
    /** Language code for Breton */
    LanguageCode["br"] = "br";
    /** Language code for Bulgarian */
    LanguageCode["bg"] = "bg";
    /** Language code for Burmese */
    LanguageCode["my"] = "my";
    /** Language code for Catalan, Valencian */
    LanguageCode["ca"] = "ca";
    /** Language code for Central Khmer */
    LanguageCode["km"] = "km";
    /** Language code for Chamorro */
    LanguageCode["ch"] = "ch";
    /** Language code for Chechen */
    LanguageCode["ce"] = "ce";
    /** Language code for Chichewa, Chewa, Nyanja */
    LanguageCode["ny"] = "ny";
    /** Language code for Chinese */
    LanguageCode["zh"] = "zh";
    /** Language code for Church Slavonic, Old Bulgarian, Old Church Slavonic */
    LanguageCode["cu"] = "cu";
    /** Language code for Chuvash */
    LanguageCode["cv"] = "cv";
    /** Language code for Cornish */
    LanguageCode["kw"] = "kw";
    /** Language code for Corsican */
    LanguageCode["co"] = "co";
    /** Language code for Cree */
    LanguageCode["cr"] = "cr";
    /** Language code for Croatian */
    LanguageCode["hr"] = "hr";
    /** Language code for Czech */
    LanguageCode["cs"] = "cs";
    /** Language code for Danish */
    LanguageCode["da"] = "da";
    /** Language code for Divehi, Dhivehi, Maldivian */
    LanguageCode["dv"] = "dv";
    /** Language code for Dutch, Flemish */
    LanguageCode["nl"] = "nl";
    /** Language code for Dzongkha */
    LanguageCode["dz"] = "dz";
    /** Language code for English */
    LanguageCode["en"] = "en";
    /** Language code for Esperanto */
    LanguageCode["eo"] = "eo";
    /** Language code for Estonian */
    LanguageCode["et"] = "et";
    /** Language code for Ewe */
    LanguageCode["ee"] = "ee";
    /** Language code for Faroese */
    LanguageCode["fo"] = "fo";
    /** Language code for Fijian */
    LanguageCode["fj"] = "fj";
    /** Language code for Finnish */
    LanguageCode["fi"] = "fi";
    /** Language code for French */
    LanguageCode["fr"] = "fr";
    /** Language code for Fulah */
    LanguageCode["ff"] = "ff";
    /** Language code for Gaelic, Scottish Gaelic */
    LanguageCode["gd"] = "gd";
    /** Language code for Galician */
    LanguageCode["gl"] = "gl";
    /** Language code for Ganda */
    LanguageCode["lg"] = "lg";
    /** Language code for Georgian */
    LanguageCode["ka"] = "ka";
    /** Language code for German */
    LanguageCode["de"] = "de";
    /** Language code for Gikuyu, Kikuyu */
    LanguageCode["ki"] = "ki";
    /** Language code for Greek (Modern) */
    LanguageCode["el"] = "el";
    /** Language code for Greenlandic, Kalaallisut */
    LanguageCode["kl"] = "kl";
    /** Language code for Guarani */
    LanguageCode["gn"] = "gn";
    /** Language code for Gujarati */
    LanguageCode["gu"] = "gu";
    /** Language code for Haitian, Haitian Creole */
    LanguageCode["ht"] = "ht";
    /** Language code for Hausa */
    LanguageCode["ha"] = "ha";
    /** Language code for Hebrew */
    LanguageCode["he"] = "he";
    /** Language code for Herero */
    LanguageCode["hz"] = "hz";
    /** Language code for Hindi */
    LanguageCode["hi"] = "hi";
    /** Language code for Hiri Motu */
    LanguageCode["ho"] = "ho";
    /** Language code for Hungarian */
    LanguageCode["hu"] = "hu";
    /** Language code for Icelandic */
    LanguageCode["is"] = "is";
    /** Language code for Ido */
    LanguageCode["io"] = "io";
    /** Language code for Igbo */
    LanguageCode["ig"] = "ig";
    /** Language code for Indonesian */
    LanguageCode["id"] = "id";
    /** Language code for Interlingua (International Auxiliary Language Association) */
    LanguageCode["ia"] = "ia";
    /** Language code for Interlingue */
    LanguageCode["ie"] = "ie";
    /** Language code for Inuktitut */
    LanguageCode["iu"] = "iu";
    /** Language code for Inupiaq */
    LanguageCode["ik"] = "ik";
    /** Language code for Irish */
    LanguageCode["ga"] = "ga";
    /** Language code for Italian */
    LanguageCode["it"] = "it";
    /** Language code for Japanese */
    LanguageCode["ja"] = "ja";
    /** Language code for Javanese */
    LanguageCode["jv"] = "jv";
    /** Language code for Kannada */
    LanguageCode["kn"] = "kn";
    /** Language code for Kanuri */
    LanguageCode["kr"] = "kr";
    /** Language code for Kashmiri */
    LanguageCode["ks"] = "ks";
    /** Language code for Kazakh */
    LanguageCode["kk"] = "kk";
    /** Language code for Kinyarwanda */
    LanguageCode["rw"] = "rw";
    /** Language code for Komi */
    LanguageCode["kv"] = "kv";
    /** Language code for Kongo */
    LanguageCode["kg"] = "kg";
    /** Language code for Korean */
    LanguageCode["ko"] = "ko";
    /** Language code for Kwanyama, Kuanyama */
    LanguageCode["kj"] = "kj";
    /** Language code for Kurdish */
    LanguageCode["ku"] = "ku";
    /** Language code for Kyrgyz */
    LanguageCode["ky"] = "ky";
    /** Language code for Lao */
    LanguageCode["lo"] = "lo";
    /** Language code for Latin */
    LanguageCode["la"] = "la";
    /** Language code for Latvian */
    LanguageCode["lv"] = "lv";
    /** Language code for Letzeburgesch, Luxembourgish */
    LanguageCode["lb"] = "lb";
    /** Language code for Limburgish, Limburgan, Limburger */
    LanguageCode["li"] = "li";
    /** Language code for Lingala */
    LanguageCode["ln"] = "ln";
    /** Language code for Lithuanian */
    LanguageCode["lt"] = "lt";
    /** Language code for Luba-Katanga */
    LanguageCode["lu"] = "lu";
    /** Language code for Macedonian */
    LanguageCode["mk"] = "mk";
    /** Language code for Malagasy */
    LanguageCode["mg"] = "mg";
    /** Language code for Malay */
    LanguageCode["ms"] = "ms";
    /** Language code for Malayalam */
    LanguageCode["ml"] = "ml";
    /** Language code for Maltese */
    LanguageCode["mt"] = "mt";
    /** Language code for Manx */
    LanguageCode["gv"] = "gv";
    /** Language code for Maori */
    LanguageCode["mi"] = "mi";
    /** Language code for Marathi */
    LanguageCode["mr"] = "mr";
    /** Language code for Marshallese */
    LanguageCode["mh"] = "mh";
    /** Language code for Moldovan, Moldavian, Romanian */
    LanguageCode["ro"] = "ro";
    /** Language code for Mongolian */
    LanguageCode["mn"] = "mn";
    /** Language code for Nauru */
    LanguageCode["na"] = "na";
    /** Language code for Navajo, Navaho */
    LanguageCode["nv"] = "nv";
    /** Language code for Northern Ndebele */
    LanguageCode["nd"] = "nd";
    /** Language code for Ndonga */
    LanguageCode["ng"] = "ng";
    /** Language code for Nepali */
    LanguageCode["ne"] = "ne";
    /** Language code for Northern Sami */
    LanguageCode["se"] = "se";
    /** Language code for Norwegian */
    LanguageCode["no"] = "no";
    /** Language code for Norwegian Bokmål */
    LanguageCode["nb"] = "nb";
    /** Language code for Norwegian Nynorsk */
    LanguageCode["nn"] = "nn";
    /** Language code for Nuosu, Sichuan Yi */
    LanguageCode["ii"] = "ii";
    /** Language code for Occitan (post 1500) */
    LanguageCode["oc"] = "oc";
    /** Language code for Ojibwa */
    LanguageCode["oj"] = "oj";
    /** Language code for Oriya */
    LanguageCode["or"] = "or";
    /** Language code for Oromo */
    LanguageCode["om"] = "om";
    /** Language code for Ossetian, Ossetic */
    LanguageCode["os"] = "os";
    /** Language code for Pali */
    LanguageCode["pi"] = "pi";
    /** Language code for Panjabi, Punjabi */
    LanguageCode["pa"] = "pa";
    /** Language code for Pashto, Pushto */
    LanguageCode["ps"] = "ps";
    /** Language code for Persian */
    LanguageCode["fa"] = "fa";
    /** Language code for Polish */
    LanguageCode["pl"] = "pl";
    /** Language code for Portuguese */
    LanguageCode["pt"] = "pt";
    /** Language code for Quechua */
    LanguageCode["qu"] = "qu";
    /** Language code for Romansh */
    LanguageCode["rm"] = "rm";
    /** Language code for Rundi */
    LanguageCode["rn"] = "rn";
    /** Language code for Russian */
    LanguageCode["ru"] = "ru";
    /** Language code for Samoan */
    LanguageCode["sm"] = "sm";
    /** Language code for Sango */
    LanguageCode["sg"] = "sg";
    /** Language code for Sanskrit */
    LanguageCode["sa"] = "sa";
    /** Language code for Sardinian */
    LanguageCode["sc"] = "sc";
    /** Language code for Serbian */
    LanguageCode["sr"] = "sr";
    /** Language code for Shona */
    LanguageCode["sn"] = "sn";
    /** Language code for Sindhi */
    LanguageCode["sd"] = "sd";
    /** Language code for Sinhala, Sinhalese */
    LanguageCode["si"] = "si";
    /** Language code for Slovak */
    LanguageCode["sk"] = "sk";
    /** Language code for Slovenian */
    LanguageCode["sl"] = "sl";
    /** Language code for Somali */
    LanguageCode["so"] = "so";
    /** Language code for Sotho, Southern */
    LanguageCode["st"] = "st";
    /** Language code for South Ndebele */
    LanguageCode["nr"] = "nr";
    /** Language code for Spanish, Castilian */
    LanguageCode["es"] = "es";
    /** Language code for Sundanese */
    LanguageCode["su"] = "su";
    /** Language code for Swahili */
    LanguageCode["sw"] = "sw";
    /** Language code for Swati */
    LanguageCode["ss"] = "ss";
    /** Language code for Swedish */
    LanguageCode["sv"] = "sv";
    /** Language code for Tagalog */
    LanguageCode["tl"] = "tl";
    /** Language code for Tahitian */
    LanguageCode["ty"] = "ty";
    /** Language code for Tajik */
    LanguageCode["tg"] = "tg";
    /** Language code for Tamil */
    LanguageCode["ta"] = "ta";
    /** Language code for Tatar */
    LanguageCode["tt"] = "tt";
    /** Language code for Telugu */
    LanguageCode["te"] = "te";
    /** Language code for Thai */
    LanguageCode["th"] = "th";
    /** Language code for Tibetan */
    LanguageCode["bo"] = "bo";
    /** Language code for Tigrinya */
    LanguageCode["ti"] = "ti";
    /** Language code for Tonga (Tonga Islands) */
    LanguageCode["to"] = "to";
    /** Language code for Tsonga */
    LanguageCode["ts"] = "ts";
    /** Language code for Tswana */
    LanguageCode["tn"] = "tn";
    /** Language code for Turkish */
    LanguageCode["tr"] = "tr";
    /** Language code for Turkmen */
    LanguageCode["tk"] = "tk";
    /** Language code for Twi */
    LanguageCode["tw"] = "tw";
    /** Language code for Uighur, Uyghur */
    LanguageCode["ug"] = "ug";
    /** Language code for Ukrainian */
    LanguageCode["uk"] = "uk";
    /** Language code for Urdu */
    LanguageCode["ur"] = "ur";
    /** Language code for Uzbek */
    LanguageCode["uz"] = "uz";
    /** Language code for Venda */
    LanguageCode["ve"] = "ve";
    /** Language code for Vietnamese */
    LanguageCode["vi"] = "vi";
    /** Language code for Volap_k */
    LanguageCode["vo"] = "vo";
    /** Language code for Walloon */
    LanguageCode["wa"] = "wa";
    /** Language code for Welsh */
    LanguageCode["cy"] = "cy";
    /** Language code for Western Frisian */
    LanguageCode["fy"] = "fy";
    /** Language code for Wolof */
    LanguageCode["wo"] = "wo";
    /** Language code for Xhosa */
    LanguageCode["xh"] = "xh";
    /** Language code for Yiddish */
    LanguageCode["yi"] = "yi";
    /** Language code for Yoruba */
    LanguageCode["yo"] = "yo";
    /** Language code for Zhuang, Chuang */
    LanguageCode["za"] = "za";
    /** Language code for Zulu */
    LanguageCode["zu"] = "zu";
})(LanguageCode || (LanguageCode = {}));
